### PR TITLE
VEN-474 | Pricing page

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Added support for a basic theme in the Table component
- Implement the pricing page with mock data
- Implement `format` utils for prices and percentages

### Closes :no_good_woman:
[VEN-474](https://helsinkisolutionoffice.atlassian.net/browse/VEN-474): Implement Pricing page

### Automated tests :gear:️
- https://github.com/City-of-Helsinki/berth-reservations-admin/blob/411805d7d60bf4133d784161f6d1c2ceb33ee1fa/src/common/utils/format.test.js#L50
- https://github.com/City-of-Helsinki/berth-reservations-admin/blob/e407658d57ee715dd4387635e3adeb2c660fa0a5/src/domain/pricing/PricingPage.test.jsx#L79

### Manual testing :construction_worker_man:
- From the sidebar, click on `Hinnasto`
- Or navigate to `/pricing`

## Screenshots :camera_flash:
image

## Additional notes :spiral_notepad:
I decided to implement the editing modals outside the scope of this ticket to keep the PR manageable

### CODE REVIEW

File: src/common/icons/IconInvoice.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default ({ className = '' }: Props) => (
   >
     <path
       d="M11 .5c6.075 0 11 4.925 11 11s-4.925 11-11 11-11-4.925-11-11S4.925.5 11 .5zm0 1.792a9.208 9.208 0 100 18.416 9.208 9.208 0 000-18.416zm.682 4.18c1.692 0 2.834 1.113 3.25 2.531l-1.773 1.14c0-1.14-.416-2.057-1.477-2.057-.927 0-1.437.723-1.679 1.724h2.539l-.376 1.32H9.842v.807h2.122l-.39 1.32h-1.57c.228.932.698 1.558 1.678 1.558 1.088 0 1.464-1.001 1.477-2.141l1.773 1.154c-.416 1.404-1.49 2.6-3.25 2.6s-3.089-1.057-3.559-3.17H6.968v-1.321h.994c-.013-.153-.013-.32-.013-.5v-.307h-.98V9.81h1.14c.47-2.197 1.894-3.337 3.573-3.337z"
-      fill-rule="nonzero"
+      fillRule="nonzero"
```
Comment (TatuArvela) [inline src/common/icons/IconInvoice.tsx:15]: Is this change intentional? This seems like an error
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill-rule

File: src/common/utils/format.test.js [hunk 2]
```diff
@@ -10,7 +16,7 @@ describe('format', () => {
       expect(formatDimension(1, 'fi')).toMatch('m');
     });
 
-    it('should return "-" if the value is undefined/null', () => {
+    it('should return null if the value is undefined/null', () => {
```
Comment (TatuArvela) [inline src/common/utils/format.test.js:19]: Would a dash still be better than showing null to the end user?

Comment (Nizar-Rahme) [reply src/common/utils/format.test.js]: True, fixed

File: src/common/icons/IconInvoice.tsx (hunk 1 above)
Comment (Nizar-Rahme) [reply src/common/icons/IconInvoice.tsx:15]: This is a `tsx` file, not an `svg`, so I replaced it with a camel-cased attribute, after getting a `React` error in the console complaining about it.

Comment (TatuArvela) [reply src/common/icons/IconInvoice.tsx:15]: Interesting, it appears React or the bundling configs of create-react-app do something to make the SVG camelCase attributes work as expected. My mistake, ignore this comment
